Userland/Utilities/shot.cpp

```diff
@@ -49,6 +49,9 @@ int main(int argc, char** argv)
     if (output_path.is_empty())
         output_path = Core::DateTime::now().to_string("screenshot-%Y-%m-%d-%H-%M-%S.png");
     
+    if (!output_path.ends_with(".png"))
+        output_path = String::formatted("{}.png", output_path);
```

Comment:
If the user didn't append `.png` we should assume the user didn't want `.png` :)

Reply:
@awesomekling  Hmmm thought it only supports PNG format..

Reply:
Yes it does, but if the user wants to create a file named `x` we should respect that.

Reply:
@awesomekling Okay, sounds fair. I like your way of thinking, like, giving the user freedom to do what he wants. Thank you for reviewing my pull request!